guard/ts-lib/index.ts

```diff
@@ -109,27 +109,50 @@ const formatOutput = ({ result, rulesNames, dataNames }: FormatOutputParams): Sa
   return JSON.parse(output);
 };
 
-async function readFiles(dirPath: string, supportedExtensions: string[]): Promise<TraversalResult> {
+async function readFiles(pathOrFile: string, supportedExtensions: string[]): Promise<TraversalResult> {
```

Comment:
nit: this should just either be path or dirOrFile (i think path is best)

Reply:
on it